api/controllers/game/create.js

```diff
@@ -1,14 +1,16 @@
 var gameAPI = sails.hooks['customgamehook'];
 
 module.exports = function (req, res) {
```

Comment:
I feel like this should be updated to async await also, but it's not a blocker by any means.

Reply:
Yeah I agree. Trying to minimize current changes but we could update the entire backend to switch I'd imagine

Reply:
Yep, tech debt 👍🏼